### ISSUE TITLE
Revoked certificates still shown as valid in CovPassCheck

### ISSUE THREAD

**Issue by treysis** [Digitaler-Impfnachweis/documentation#28]
## Avoid duplicates
* [ ] Bug is not mentioned in the [FAQ](https://digitaler-impfnachweis-app.de/faq/)
* [ ] Bug affects both Android **and** iOS, for specific issues / questions that apply only to one operating system, please raise them in the respective repositories:
    * [Android repository](https://github.com/Digitaler-Impfnachweis/covpass-android)
    * [iOS repository](https://github.com/Digitaler-Impfnachweis/covpass-ios)
* [ ] Bug is not already reported in another issue

## Technical details

- Device name: irrelevant
- OS version: irrelevant
- App version: 1.20

## Describe the bug

The fraudulent QR codes that leaked 1-2 weeks ago are still validated by CovPassCheck. Other check apps (e.g. the Swiss Covid Certificate Check) correctly mark them as revoked. The database of CovPassCheck says it has been updated today.

## Expected behaviour

The QR codes should be reported as invalid.

**Comment by alexcimander** [comment]
Hey Treysis,

sorry for the late response! Is this still an issue? With the Release of 1.13 the certificates should be blocked :-) 

Thank you for your feedback!

**Comment by treysis** [comment]
Hello @alexcimander 
Unfortunately, certificates are still accepted both by CovPass and CovPassCheck.
E.g. this certificate for Adolf Hitler:
https://twitter.com/reversebrain/status/1453095038284615682

**Comment by alexcimander** [comment]
Hey Treysis, 

the CovPass blocked certificates from entities that we were advised to block. There are still some that will be accepted. The Adolf Hitler certificate is one of the ones that are still scanable. However we are working on a further prevention and we might block more certificates that were issued erroneously. We will keep you updated 👍

**Comment by treysis** [comment]
Thx. Why is it so slow, though? Those certificates leaked end of October. Italy and Switzerland are already blocking them since shortly after.

**Comment by OlympianRevolution** [comment]
Yes the Mickey mouse certificate at least is still valid in the CoronaWarnApp.

We were told to report it here https://github.com/corona-warn-app/cwa-documentation/issues/751

For more Certs to revoke see here https://github.com/denysvitali/covid-cert-analysis/blob/master/RESULTS.md#samplesmickeymousetxt

**Comment by OlympianRevolution** [comment]
Mickey Mouse has also been revoked by Tous Anti COVID (France). What is the rational for delaying revocations already performed by other countries?
